documentation/src/test/java/example/testrule/StopwatchVintageDemo.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package example.testrule;
+
+// tag::user_guide[]
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+import org.junit.AssumptionViolatedException;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Stopwatch;
+import org.junit.runner.Description;
+
+public class StopwatchVintageDemo {
+
+	private static final Logger logger = Logger.getLogger("");
```

Comment:
The code in the vintage demonstrations was copied directly from the JUnit 4 javadocs - in this case it's at: http://junit.org/junit4/javadoc/latest/org/junit/rules/Stopwatch.html.  I made changes to the JUnit 4 sample code when it wouldn't run as shown, and in this case combined both the examples in the javadocs into one file.  Do you think it's worth the effort to turn the vintage examples into production-ready code?  I have to admit that I completely missed that the code was logging to the root logger ... and it does feel a bit dirty!